docs/service-brokers/gcp-broker-service-classes/cloud-sql-mysql/docs/plans-details.md

```diff
@@ -0,0 +1,133 @@
+| **masterHeartbeatPeriod** | `string` | Interval in milliseconds between replication heartbeats. | NO | - |
+| **password** | `string` | The password for the replication connection. | NO | - |
+| **sslCipher** | `string` | A list of permissible ciphers to use for SSL encryption. | NO | - |
+| **username** | `string` | The username for the replication connection. | NO | - |
+| **verifyServerCertificate** | `boolean` | Whether or not to check the master's Common Name value in the certificate that it sends during the SSL handshake. | NO | - |
+
+### Settings properties
+
+These are the **Settings** properties:
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| **activationPolicy** | `string` | The activation policy specifies when the instance is activated. It is applicable only when the instance state is `RUNNABLE`. The possible values are `ALWAYS`, `NEVER` and `ON_DEMAND`. `ALWAYS` indicates that the instance is on, and remains so even in the absence of connection requests. `NEVER` means that the instance is off and is not activated, even if a connection request arrives. `ON_DEMAND` applies to First Generation instances only and it indicates that the instance responds to incoming requests, and turns itself off when not in use. Instances with `PER_USE` pricing turn off after 15 minutes of inactivity. Instances with `PER_PACKAGE` pricing turn off after 12 hours of inactivity. | NO | - |
+| **authorizedGaeApplications** | `string` | The App Engine application IDs that can access this instance. This property is only applicable to First Generation instances. | NO | - |
+| **backupConfiguration** | `string` | The daily backup configuration for the instance. For more information, go to the **BackupConfiguration properties** section. | NO | - |
+| **crashSafeReplicationEnabled** | `boolean` | Configuration specific to read replica instances. Indicates whether database flags for crash-safe replication are enabled. This property is only applicable to First Generation instances. | NO | - |
+| **dataDiskSizeGb** | `string` | The size of data disk in `GB`. The data disk size minimum is `10GB`. Applies only to Second Generation instances. | NO | - |
+| **dataDiskType** | `string` | The type of data disk. The possible values are `PD_SSD`, `PD_HDD`. Applies only to Second Generation instances. | NO | `PD_SSD` |
+| **databaseFlags** | `array` | The database flags passed to the instance at startup. | NO | - |
+| **databaseFlags.name** | `string` | The name of the flag. These flags are passed at instance startup, so include both MySQL server options and MySQL system variables. Flags should be specified with underscores, not hyphens. For more information, see **Configuring MySQL Flags** in the Google Cloud SQL documentation, as well as the official MySQL documentation for server options and system variables. | NO | - |
```

Comment:
Wondering if there is a link to google documentation that we can put here? or it's not recommended :) To point to
`**Configuring MySQL Flags** in the Google Cloud SQL documentation`

Reply:
I've deleted the reference.